topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga

```diff
@@ -0,0 +1,161 @@
+{
+  "a_galaxy_workflow": "true",
```

Comment:
If nothing needs to be translated here, could probably symlink the folder.

Reply:
Symlink the folder? What does this mean?

Reply:
OH as in nab the workflow link from the original? In which case... yes!

Reply:
Oh, it's a symbolic link, a pointer to the folder that still behaves like a folder. I can do it tomorrow! :)

Reply:
Converted workflows folder to symlink. https://github.com/galaxyproject/training-material/pull/2534/files#diff-57f27e497983460cff1807a3e1b5ee8cda56494146cceaf8ead3b985737040a3 That way we can ensure the WF stays in sync with the english version.

Reply:
Ace!